lib/api.ts

```diff
@@ -51,11 +54,19 @@ export class APIClient {
     return this.request.post(`${this.apiRoot}/send/email`, req.message);
   }
 
+  getCustomersByEmail(email: string) {
```

Comment:
Rather than add this to the `APIClient`, a new `BetaAPIClient` should be created for this. This would mirror how [our api docs](https://customer.io/docs/api/#section/Overview) are set up, and keep the two apis (app vs. beta) separate from each other, which they are